modules/ROOT/pages/ip-allowlist.adoc

```diff
@@ -0,0 +1,72 @@
+= IP Allowlist
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: IP, allowlist, validation, policy
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | IP Allowlist
+>s|Summary      | Allows a list or range of specified IP addresses to request access   
+>s|Category | Security
+>s|First Mule version available | v3.8.0
+.1+>.^s| Returned Status Codes
+| 403 - IP is rejected
+|===
+
+
+The IP Allowlist policy allows a list or a range of specified IP addresses to access a configured API endpoint.
+
+The IP Allowlist policy allows access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
+
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule) or MEL expression for Mule 3. You specify the expression to define the source IP to be used when the policy tries to determine whether the IP belongs to the restricted IP list defined in the policy.
+
+
+[[specifying_ip_address]]
+== Specifying IP Addresses to Allowlist in Your Policy 
+
+You can specify the IP addresses to approve in the policy in the following ways:
+
+* Allowlist access based on the IP address of the request.
+* Allowlist access based on the origin IP address that is determined by resolving a specific DataWeave or MEL expression, such as the https://en.wikipedia.org/wiki/X-Forwarded-For[x-Forwarded-For] header.
+* Any other origin.
```

Comment:
I don't understand this list item; it does not logically complete the list intro and is not parallel to the other items. 

The list is introduced as _ways_ to allowlist addresses, but Lines 28-32 seem to be saying that you can allowlist two _types_ of addresses: either request IP addresses or origin IP addresses, including that determined by resolving.....